### DISCUSSION TITLE
Fast5 playback error

### DISCUSSION THREAD

**Discussion by dmac83** [LooseLab/readfish#246] [Q&A]
Hi, we are able to load and run your bulk FAST5 for playback using your configurations. However, when we try to run our bulk FAST5 it fails and we get an error message “Unable to open HDF file'. We have a Mk1c device - could that be the issue? Thanks!

**Comment by Adoni5** [comment]
Hi @dmac83 - I'm not sure I'm clear on where the error is -

> Hi, we are able to load and run your bulk FAST5 for playback using your configurations
> However, when we try to run our bulk FAST5 it fails and we get an error message “Unable to open HDF file'.

Is the error when you try to run with readfish?

**Comment by Adoni5** [comment]
One thing I would check is that the FAST5 file isn't corrupt - can you open it in [HDFView](https://www.hdfgroup.org/downloads/hdfview/)
